README.DockerHub.md

```diff
@@ -4,11 +4,11 @@
 
 # Supported Windows Server 2016 Version 1709 (Fall Creators Update) amd64 tags
 
-- [`dotnetapp-nanoserver-1709` (*dotnetapp-prod/Dockerfile*)](https://github.com/dotnet/dotnet-docker-samples/blob/master/dotnetapp-prod/Dockerfile)
+- [`dotnetapp-nanoserver-1709`, `dotnetapp`, `latest` (*dotnetapp-prod/Dockerfile*)](https://github.com/dotnet/dotnet-docker-samples/blob/master/dotnetapp-prod/Dockerfile)
```

Comment:
We haven't advertised the mulit-arch tags yet since it is only supported on edge versions of Docker.  Please revert this change.